RoddyCore/src/de/dkfz/roddy/Roddy.java

```diff
@@ -618,17 +618,17 @@ private static void setDefaultRoddyQueueVariable(RecursiveOverridableMapContaine
      * @return
      */
     private static void setScratchDirectory(RecursiveOverridableMapContainerForConfigurationValues configurationValues) {
-        String deprecatedDefaultScratchDir = Roddy.getApplicationProperty(ConfigurationConstants.CVALUE_DEFAULT_SCRATCH_DIR, "");
+        String deprecatedDefaultScratchDir = Roddy.getOrSetApplicationProperty(ConfigurationConstants.CVALUE_DEFAULT_SCRATCH_DIR, "");
         if (!deprecatedDefaultScratchDir.equals("")) {
```

Comment:
If this leads to a deprecation warning, you might also put it in a separate method and mark this as deprecated as well. This way, we'll also get a deprecation warning on compile time.

Reply:
done